### ISSUE TITLE
Newbie Question about the UI

### ISSUE THREAD

**Issue by acf** [keirf/HxC_FF_File_Selector#6]
Hi Keir, 

I just got a little gotek and your excellent FlashFloppy software for the first time.  It's in the form of an external dongle with selector knob and little screen, all very nice and works absolutely fine with my A1200 so far (in that disks in slots are loaded happily when I turn the selection dial).

I have a little problem with the Selector software  (v9-FF according to the screen) however.

I can navigate into a subfolder ok but once I'm in to the subfolder, I can't ever seem to get back up a top level again.  For the moment I allocate all the floppies in subdir X, save the config, reboot, navigate into subdir Y... rinse, repeat.

The help menu says "right shift : Go back to the top of the folder" and I assumed that meant "go up" but it doesn't seem t do anything for me.  I'm using the keyboard as I don't have a joystick attached (I don't really play games on the Amiga).

In short, how do I `CD ..`  ?

Any help appreciated, and I realise this is probably not the right forum for user questions, but YouTube and google failed me :-)

**Comment by keirf** [comment]
Just don't use it. Delete HXCSDFE.CFG and AUTOBOOT.HFE from your USB drive and instead you will be able to directly navigate among all your image files directly using your external dongle. The hosted selector program is fairly horrible.

**Comment by alanflex** [comment]
Aha!  I thought it was compulsory \o/. 

I was even wondering whether I might write a little Mac app to help manage the slots myself because I figured it was just writing a slot=>filename mapping config file, but then I realised it wasn't and was resigned to figuring it out :-)

Perfect, thanks so much.  I don't have many ADFs to wrangle, it's just various versions of workbench and some utility programs for an A500 without an HD.

(oops, wrong GitHub account)
